package.json

```diff
@@ -201,8 +201,8 @@
   "peerDependencies": {
     "moment": "2.x",
     "moment-timezone": "0.5.x",
-    "react": "16.x",
-    "react-dom": "16.x",
+    "react": ">= 16.8.0",
```

Comment:
I wasn't 100% sure on this, but this is the same syntax that `styled-components` are using on their `5.0.0` branch 🤷‍♂

Reply:
Yeah we use the same syntax on react-router and other deps I think.